api/comments/serializers.py

```diff
@@ -26,17 +28,19 @@ class CommentSerializer(JSONAPISerializer):
         'deleted',
         'date_created',
         'date_modified',
+        'page',
         'target'
     ])
 
     id = IDField(source='_id', read_only=True)
     type = TypeField()
     content = AuthorizedCharField(source='get_content', required=True, max_length=osf_settings.COMMENT_MAXLENGTH)
+    page = ser.CharField(read_only=True)
```

Comment:
The `page` field is primarily used to make filtering easier. In the discussions page (which is not a part of this PR), users have to option to view all comments on the project overview page, all comments on files, etc.